container/rockylinux-9-dev/Dockerfile

```diff
@@ -18,6 +18,8 @@ COPY scripts/. ./
 RUN \
   sed -i '/^\[crb\]$/,/^\[/s#^enabled=0$#enabled=1#' /etc/yum.repos.d/rocky.repo \
   && grep '^enabled=1$' /etc/yum.repos.d/rocky.repo \
+  && sed -i '/^\[devel\]$/,/^\[/s#^enabled=0$#enabled=1#' /etc/yum.repos.d/rocky-devel.repo \
+  && grep '^enabled=1$' /etc/yum.repos.d/rocky-devel.repo \
```

Comment:
A quick question, what does the path `/etc/yum.repos.d/rocky-devel.repo` refer to? I guess I'm wondering where does that path point to?

Reply:
I see, does this path only exist in RHEL then? If I go on an RHEL machine and do `ls /etc/` then I will see `yum.repos.d`?

Reply:
Got it, thanks!